src/nixpacks/app.rs

```diff
@@ -171,10 +171,7 @@ impl App {
 
     /// Convert an absolute path to a path relative to the app source directory
     pub fn strip_source_path(&self, abs_path: &Path) -> Result<PathBuf> {
-        let source_str = match self.source.to_str() {
-            Some(s) => s,
-            None => bail!("Failed to parse source path"),
-        };
+        let source_str = self.source.to_str().unwrap_or("");
```

Comment:
Why default to empty string here?

Reply:
Oops, got left in.

Reply:
Fixed in [`c236391` (#507)](https://github.com/railwayapp/nixpacks/pull/507/commits/c2363914f6b5e09ed8ad050499fd5a29d492a59b)